src/components/postView/container/postDisplayContainer.tsx

```
@@ -34,43 +34,52 @@ const PostDisplayContainer = ({
+        prevCount !== activeVotesCount ? activeVotesCount : prevCount,
+      );
     }
   }, [post]);
 
   useEffect(() => {
-    _fetchPost();
-  }, [isFetchPost, isFetchComments]);
+    if (isFetchPost || isFetchComments) {
+      _fetchPost();
+    }
+  }, [isFetchPost, isFetchComments, _fetchPost]);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)